app/models/user.rb

```diff
@@ -1,17 +1,21 @@
 class User < ApplicationRecord
-  attr_accessor :remember_token
+  attr_accessor :remember_token, :activation_token
   USER_ATTR = %i(name email password password_confirmation).freeze
   VALID_EMAIL_REGEX = Settings.validates_email_regex
 
-  before_save{email.downcase!}
+  scope :where_activated, ->{where activated: true}
```

Comment:
```suggestion
  scope :activated, ->{where activated: true}
```
đặt tên rõ mục đích của scope là được rồi